OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs

```diff
@@ -167,7 +167,7 @@ public override void DrawOuter()
 			Game.Renderer.EnableScissor(drawBounds);
 
 			drawBounds.Offset((-ChildOrigin).ToPoint());
-			foreach (var child in Children)
+			foreach (var child in Children.ToArray())
```

Comment:
Manipulating the UI from an external thread is a very bad idea, and will (and has in the past) cause(d) a bunch of hard to find regressions.